platform/node/src/node_map.cpp

```diff
@@ -278,9 +278,8 @@ void NodeMap::renderFinished() {
             [](char *, void *hint) {
                 delete reinterpret_cast<const mbgl::StillImage *>(hint);
             },
-            const_cast<mbgl::StillImage *>(img.get())
+            const_cast<mbgl::StillImage *>(img.release())
         ).ToLocalChecked();
-        img.release();
```

Comment:
AFIK, there are no cross-platform guarantees about the order in which arguments are evaluated